assets/css/phoenix.css

```diff
@@ -6,78 +6,113 @@
 
-/* General style */
-h1{font-size: 3.6rem; line-height: 1.25}
-h2{font-size: 2.8rem; line-height: 1.3}
-h3{font-size: 2.2rem; letter-spacing: -.08rem; line-height: 1.35}
-h4{font-size: 1.8rem; letter-spacing: -.05rem; line-height: 1.5}
-h5{font-size: 1.6rem; letter-spacing: 0; line-height: 1.4}
-h6{font-size: 1.4rem; letter-spacing: 0; line-height: 1.2}
-pre{padding: 1em;}
+ /* General style */
+ h1{font-size: 3.6rem; line-height: 1.25}
+ h2{font-size: 2.8rem; line-height: 1.3}
+ h3{font-size: 2.2rem; letter-spacing: -.08rem; line-height: 1.35}
+ h4{font-size: 1.8rem; letter-spacing: -.05rem; line-height: 1.5}
+ h5{font-size: 1.6rem; letter-spacing: 0; line-height: 1.4}
+ h6{font-size: 1.4rem; letter-spacing: 0; line-height: 1.2}
+ pre{padding: 1em;}
+ 
+ body{
+   font-family: "Gill Sans", sans-serif;
+ }
 
-.container{
-  margin: 0 auto;
-  max-width: 80.0rem;
-  padding: 0 2.0rem;
-  position: relative;
-  width: 100%
-}
-select {
-  width: auto;
-}
-
-#entry{
-  background-color: aliceblue;
-  padding: 10px;
-  border: 2px solid darkblue;
-}
+ h1, h2, h3, h4, h5, h6{
+  font-family: "PT Serif", serif;
+ }
+ 
+ .container{
+   margin: 0 auto;
+   max-width: 80.0rem;
+   padding: 0 2.0rem;
+   position: relative;
+   width: 100%
+ }
+ select {
+   width: auto;
+ }
 
-/* Headers */
-header {
-  width: 100%;
-  background: #fdfdfd;
-  border-bottom: 1px solid #eaeaea;
-  margin-bottom: 2rem;
-}
-header section {
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-}
-header section :first-child {
-  order: 2;
-}
-header section :last-child {
-  order: 1;
-}
-header nav ul,
-header nav li {
-  margin: 0;
-  padding: 0;
-  display: block;
-  text-align: right;
-  white-space: nowrap;
-}
-header nav ul {
-  margin: 1rem;
-  margin-top: 0;
-}
-header nav a {
-  display: block;
-}
+ a {
+   color: #344968;
+   text-decoration: underline;
+ }
+ 
+ #entry{
+   background-color: aliceblue;
+   padding: 10px;
+   border: 2px solid #1d3557;
+ }
+ 
+ #title{
+   text-align: center;
+   font-weight: bold;
+   color: #1d3557;
+ }
+ 
```

Comment:
You might want to define a separate class for metadata fields like published date and author.